hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -29,6 +29,8 @@
 from hmalib.common.logging import get_logger
 from hmalib.common.s3_adapters import S3ThreatDataConfig
 from hmalib.common.signal_models import PDQSignalMetadata
+from hmalib.common.fetcher_modules import ThreatExchangeConfig
```

Comment:
I think there are some other places that we need to import ThreatExchangeConfig. For example, in populate_config_db.py, line 19 "from hmalib.lambdas.fetcher import ThreatExchangeConfig". Do we need to change all of them?

Reply:
Good call out. I wonder why mypy didn't realize that